daemon/cluster/executor/container/container.go

```diff
@@ -664,11 +665,8 @@ func networkCreateRequest(name string, nw *api.Network) clustertypes.NetworkCrea
 	}
 
 	return clustertypes.NetworkCreateRequest{
-		ID: nw.ID,
-		CreateRequest: network.CreateRequest{
-			Name:          name, // TODO(thaJeztah): this is the same as [nw.Spec.Annotations.Name]; consider using that instead
-			CreateOptions: options,
-		},
+		ID:            nw.ID,
+		CreateRequest: req,
```

Comment:
Not for this PR, but now starting to wonder if we even need the intermediate `CreateRequest` here if it's gonna be set for the `clustertypes.NetworkCreateRequest`, but _POSSIBLY_ we (or I?) did so to make sure we don't forget updating things in the cluster types when adding fields to the moby API etc.